resources/env/authoring/tomcat-config/crafter/studio/extension/studio-config-override.yaml

```diff
@@ -27,7 +27,7 @@ studio.configuration.site.defaultAuthoringUrl: ^https?://localhost:8080/studio/?
 # Default preview deployer URL (can be overridden per site)
 studio.preview.defaultPreviewDeployerUrl: ${env:DEPLOYER_URL}/api/1/target/deploy/{siteEnv}/{siteName}
 # Default preview create target URL (can be overridden per site)
-studio.preview.createTargetUrl: ${env:DEPLOYER_URL}/api/1/target/upsert
+studio.preview.createTargetUrl: ${env:DEPLOYER_URL}/api/1/target/create-if-not-exists
```

Comment:
We're using `_` instead of `-` in our URLs now.

Reply:
Deployer controllers have a lot of `-` in their URLs. Should we change them or just this one?